ouroboros-consensus/changelog.d/20240724_170351_alexander.esgen_loe_chaindb_qsm.md

```diff
@@ -0,0 +1,3 @@
+### Breaking
+
+- ChainDB: allow to trigger chain selection synchronously
```

Comment:
Should we update the PR description accordingly?

Reply:
... and maybe provide a rationale on how this change relates to adding LoE to the ChainDB tests.

Reply:
Good point, updated the PR description. The reason for adding this is just testing (also see the first commit message).